Sources/ReactorKit/IdentityEquatable.swift

```diff
@@ -8,6 +8,8 @@
 public protocol IdentityEquatable: AnyObject, Equatable {}
 
 extension IdentityEquatable {
+  @inlinable
+  @inline(__always)
```

Comment:
For functions that need to be captured in the stack trace, it is appropriate not to use inline.

Reply:
I didn't know that. Thank you 🙏